.travis.yml

```diff
@@ -0,0 +1,51 @@
+language: cpp
+cache:
+  apt: true
+
+matrix:
+  include:
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+            - boost-latest
+            - sourceline: 'ppa:allegro/5.2'
+          packages:
+            - g++-5
+            - liballegro5-dev
+            - liballegro-acodec5-dev
+            - liballegro-audio5-dev
+            - liballegro-dialog5-dev
+            - liballegro-image5-dev
+            - liballegro-ttf5-dev
+            - liballegro-video5-dev
+            - libboost1.55-all-dev
+      env:
+         - MATRIX_EVAL="CC=gcc-5 && CXX=g++-5"
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - llvm-toolchain-trusty-4.0
+            - boost-latest
+            - sourceline: 'ppa:allegro/5.2'
+          packages:
+            - clang-4.0
+            - liballegro5-dev
+            - liballegro-acodec5-dev
+            - liballegro-audio5-dev
+            - liballegro-dialog5-dev
+            - liballegro-image5-dev
+            - liballegro-ttf5-dev
+            - liballegro-video5-dev
+            - libboost1.55-all-dev
+      env:
+        - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
+        
+before_install:
+    - eval "${MATRIX_EVAL}"
+install: true
+
+script: mkdir travis-build && cd travis-build && cmake -DTEST=YES -DCMAKE_BUILD_TYPE=Release .. && make
```

Comment:
-DTEST flag should be -DWANT_TEST in case of runtime only build.
Name of this flag was changed in commit [https://github.com/cb-hackers/cbEnchanted/commit/4f30d1761f2b262fc7afd7fe93a7e16adf53fee4](https://github.com/cb-hackers/cbEnchanted/commit/4f30d1761f2b262fc7afd7fe93a7e16adf53fee4).